README.md

```diff
@@ -141,6 +142,11 @@ Valid options:
 - `SMALL2BIG`
 
 
+### `EXCLUDE_FILE` variable
+
+This variable is optional.  It allows you to exclude a file (e.g. index.html) from the file listings.
```

Comment:
I figured a single file was sufficient for the primary / most common need of just excluding the index / error file that is required for this project to function.

I agree that a regex would be nicer / more featureful, but I can't think of any need I'd have for that additional feature so I didn't add it.   (this project requires that a new / single file exist and I just wanted to exclude that from the listings)